### PR TITLE
Added requirements.txt and Updated README.rst

### PR DESCRIPTION
**Pull request by AbhishekSrikanth**
### This solves issue #12 
- Added requirements.txt
- Added instructions to set up a virtual environment and to install dependencies.
- Added instructions for opening example notebook.
- Fixed syntax errors in /test_data/demo.py

### CODE REVIEW

**Comment by reckart** [comment]
Thanks ! :)

I understand that according to your instructions, a virtualenv named `pydkpro` is created - i.e. that would create the environment *inside* the `pydkpro` folder that is already part of the repository. Is that intentional?

File: README.rst [hunk 1]
```diff
@@ -42,6 +50,12 @@ Features
 Usage
 -----
 
+**How to open examples notebook**
+
+    $ cd /Examples
```
Comment (aggarwalpiush) [inline README.rst]: Directory ```Examples``` is relative. So, ```cd Examples``` should be used.

**Comment by AbhishekSrikanth** [comment]
Thanks for the feedback. I'll commit suggested changes.

**Comment by AbhishekSrikanth** [comment]
I've added changes. Please review conda instructions. I don't use conda very much. Consider adding environment.yml containing dependencies.

File: README.rst [hunk 2]
```diff
@@ -28,11 +28,36 @@ System requirements
...
+**Creating virtual environment**
+
+Install virtualenv if not already installed.
+
+    $ python -m pip install virtualenv 
+
 Create virtual environment.
 
-    $ virtualenv -p python3 pydkpro
+    $ virtualenv -p python3 \path\to\repo
```
[the remark this answers is not in the document]
Comment (aggarwalpiush) [reply README.rst]: Also, I am not sure about ```\path\to\repo``` here. Wouldn't be good with some user directory like:

```mkdir [env_name] ```
``` virtualenv -p python3 [env_name]``` or  ``` python3 -m venv [env_name]```

To activate the environment:

On Windows, run:

```[env_name]\Scripts\activate.bat```

On Unix or MacOS, run:

```source [env_name]/bin/activate```

[the remark this answers is not in the document]
Comment (AbhishekSrikanth) [reply README.rst]: I had added instructions for activating virtual environment for Windows/UNIX. Although your suggestion if giving the user the choice of name and dir is good. I'll add the changes.

**Comment by AbhishekSrikanth** [comment]
I've added your suggested changes. Please verify.

**Comment by aggarwalpiush** [comment]
Readme and requirements.txt file is required to provide installation instructions to repo users

**Pull request merged**